lib/org/chromium/webidl/Parser.js

```diff
@@ -525,7 +525,7 @@ foam.CLASS({
           if (v === ';') return parser.Serializer.create();
           return parser.Operation.isInstance(v) ?
               parser.Serializer.create({operation: v}) :
-          parser.Serializer.create({pattern: v});
+              parser.Serializer.create({pattern: v});
```

Comment:
nit: If we apply this formatting change here, we should apply it to the other ones around it for consistency.